.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: weekly
```

Comment:
No reason to not also make this daily? :)
```suggestion
    interval: daily
```

Reply:
Depends how often Jinja updates, but sure